docs/openapi/schemas.yaml

```diff
@@ -415,14 +415,58 @@ Audit:
     previousAuditResult:
       someProperty: 'somePreviousValue'
 AuditResult:
-  type: object
-  description: |
-    The result of an audit, which is dependent on the audit type.
-  additionalProperties: true
+  description: The result of an audit, which is dependent on the audit type.
+  oneOf:
+    - type: object
+      additionalProperties: true
+    - type: object
+      properties:
+        experimentationOpportunities:
+          type: array
+          description: A list of experimentation opportunities
+          items:
+            oneOf:
+              - $ref: './schemas.yaml#/ExpOpportunityGeneric'
 AuditList:
   type: array
   items:
     $ref: './schemas.yaml#/Audit'
+ExpOpportunityGeneric:
```

Comment:
opportunity can be with out experimentation as well right? like broken back links, translation etc. can we make the opportunity generic?